README.md

```diff
@@ -116,6 +125,8 @@ __Build a distribution__
 
 You can find the distribution zip at `build/distributions`.
 
+To run or debug Verify Service Provider from an IDE, you will need to either delegate the run action to gradle or set a run configuration which uses the arguments ```server configuration/verify-service-provider.yml``` to provide the configuration file.
```

Comment:
Is this true? Is that because the file is in `configuration` when you run it from the IDE?